### DISCUSSION TITLE
Creating a pr from a branch in a forked repo to the original repo

### DISCUSSION THREAD

**Discussion by wasabbe** [pwntester/octo.nvim#543] [Q&A]
Let's say I have repo A and I fork it. Now on my forked repo I create a branch called my-branch. I want to create a pr to the main branch of repo A from my-branch. How can I accomplish this using octo.nvim ? I saw that this can be done through the github cli because it prompts you in which repo to create the pr to, but when I try to do this from the octo.nvim it does not behave the same way. I love this tool so much, I use it everyday!!! this is the first time I couldn't figure out the behavior from just reading the docs.

**Comment by iton0** [comment]
@wasabbe did you ever figure out how to do this?

**Reply by wasabbe** [reply]
No I did not !

**Reply by wasabbe** [reply]
I created an issue from this discussion.
